pod.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: scheduler-pod-healthcheck
+spec:
+  containers:
+  - name: pause
+    image: gcr.io/google_containers/pause:3.0.0
```

Comment:
This isn't going to work in our environments without Internet.

We'll need to either hardcode this to our mirror names, or make it a parameter

Reply:
What have we done elsewhere/which would you recommend? I had it hard coded but Stuart pointed out that won't work in open source.

Reply:
wondering if it'd be better to merge this into the script rather than having a separate asset

Reply:
Mhm, that should be fairly easy.
Just write it out to `/tmp` using heredocs - environment variable interpolation will just work

Reply:
any example of that? googling heredocs mostly just comes up with a wikipedia entry, not sure if that's a bash tool or

Reply:
nvm, think I got it